test/e2e/util.go

```diff
@@ -728,14 +728,14 @@ func runKubectl(args ...string) string {
 }
 
 // testContainerOutput runs testContainerOutputInNamespace with the default namespace.
-func testContainerOutput(scenarioName string, c *client.Client, pod *api.Pod, expectedOutput []string) {
-	testContainerOutputInNamespace(scenarioName, c, pod, expectedOutput, api.NamespaceDefault)
+func testContainerOutput(scenarioName string, c *client.Client, pod *api.Pod, containerId int, expectedOutput []string) {
```

Comment:
I think this test API change should be split into a separate PR -- it has merit on its own.